backends/tfhe-cuda-backend/cuda/src/integer/integer.cu

```diff
@@ -338,3 +336,20 @@ void cuda_integer_reverse_blocks_64_inplace(void *const *streams,
   host_radix_blocks_reverse_inplace<uint64_t>((cudaStream_t *)(streams),
                                               gpu_indexes, lwe_array);
 }
+
+void reverseArray(uint64_t arr[], size_t n) {
```

Comment:
I also took the opportunity to update degrees after reversing blocks but it does not affect the test result.